### ISSUE TITLE
Confused about parameter settings

### ISSUE THREAD

**Issue by Plaitkol** [michaeltrs/DeepSatModels#19]
Good work! However, I am confused about the parameter settings on the T31TFM_1618 dataset. I tried these parameters:
  architecture: TSViT
  batch_size: 16
  dim: 256
  dim_head: 64
  dropout: 0.0
  emb_dropout: 0.0
  heads: 4
  ignore_background: true
  img_res: 48
  max_seq_len: 16
  num_channels: 14
  num_classes: 20
  num_features: 16
  patch_size: 2
  pool: cls
  scale_dim: 4
  spatial_depth: 4
  temporal_depth: 4
  class_weights: null
  loss_function: masked_cross_entropy
  lr_base: 1e-3
  lr_min: 5e-6
  lr_scheduler: cosine
  lr_start: 1e-8
  num_cycles: 1
  num_epochs: 100
  num_warmup_epochs: 10
  reset_lr: true
  steps: (0, 80000)
  weight_decay: 0.0

But the IoU is only 62%, not 63.1% presented in the paper[ViTs for SITS]. Have I missed any parameters? Do the parameters mentioned above need to be adjusted?
